framework.go

```diff
@@ -66,6 +68,85 @@ func (f *Framework) WC(clusterName string) (*client.Client, error) {
 	return c, nil
 }
 
+// LoadCluster will construct a Cluster struct using a Workload Cluster's
+// cluster and default-apps App CRs on the targeted Management Cluster. The
+// name and namespace where the cluster are installed need to be provided with
+// the E2E_WC_NAME and E2E_WC_NAMESPACE env vars.
+//
+// If one of the env vars are not set, a nil Cluster and error will be
```

Comment:
```suggestion
// If one of the env vars are not set, a nil Cluster and nil error will be
```

I first misunderstood this and thought it meant it'd return `nil, err`